develop-image/Dockerfile

```diff
@@ -57,8 +57,7 @@ RUN sudo make clean
 
 RUN pipenv --python python3.8
 RUN pipenv run pip3 install --upgrade pip
-RUN pipenv install --dev --skip-lock
-
-RUN pipenv run pip3 install .[all]
+RUN pipenv install --dev --skip-lock --system
 
+RUN python3 setup.py install 
```

Comment:
I don't know the context here but why are we installing using `setup.py` instead of using pip.

Reply:
https://stackoverflow.com/questions/15724093/difference-between-python-setup-py-install-and-pip-install

From what I understand from the highly detailed response to this issue,

if we use pip install, it will get packages from pypi

where as if we use this python setup.py install method, this will force the application to install from the repository. 

This then means that you can build a dev image like so;

```
docker build . \                                                                                                                                                     
-t  valory/open-aea-user:dev \
--file develop-image/Dockerfile
```

Then, the dev image can be used in the consensus repository like so;
![image](https://user-images.githubusercontent.com/35799987/154331434-d73bc889-adc1-455f-8240-62907dd62365.png)